### PR TITLE
use USE_GOOGLE_TESTS in addition to USE_CATCH_TESTS

### PR DESCRIPTION
**Pull request by jsteemann**
We need both defines, `USE_CATCH_TESTS` is set for ArangoDB versions up to including 3.4, `USE_GOOGLE_TESTS` is used from 3.5 onwards.

### CODE REVIEW

**Comment by jsteemann** [comment]
https://jenkins01.arangodb.biz/job/arangodb-matrix-pr-linux/6097/

File: scripts/buildMacOs.fish [hunk 1]
```diff
@@ -46,7 +46,8 @@ set -g FULLARGS $argv \
 
 if test "$MAINTAINER" != "On"
   set -g FULLARGS $FULLARGS \
-    -DUSE_CATCH_TESTS=Off
+    -DnUSE_CATCH_TESTS=Off \
```
Comment (KVS85) [inline scripts/buildMacOs.fish]: Typo?

File: scripts/buildMacOs.fish [hunk 2]
```diff
@@ -46,7 +46,8 @@ set -g FULLARGS $argv \
...
+    -DnUSE_GOOGLE_TESTS=Off
```
Comment (KVS85) [inline scripts/buildMacOs.fish]: Typo?

File: scripts/buildMacOs.fish (hunk 1 above)
Comment (jsteemann) [inline scripts/buildMacOs.fish]:
```suggestion
    -DUSE_CATCH_TESTS=Off \
```

File: scripts/buildMacOs.fish (hunk 2 above)
Comment (jsteemann) [reply scripts/buildMacOs.fish]: LOL, yes. Thanks for spotting it!

**Comment by mchacki** [comment]
There has been a typo in main branch where the test suffered from, try again here:
http://jenkins.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr-linux/6101/

File: scripts/buildMacOs.fish (hunk 2 above)
Comment (fceller) [reply scripts/buildMacOs.fish]: @jsteemann has this been fixed?

Comment (jsteemann) [reply scripts/buildMacOs.fish]: Yes

**Pull request merged**